defaults/main.yml

```diff
@@ -5,6 +5,7 @@
 #  - qpid-dispatch-docs
 dispatch_package_source_epel: false
 dispatch_package_pip: false
+dispatch_default_repositories: ''
```

Comment:
I'm maybe for dispatch_rhel_default_repositories: '' because these repositories are RHEL specific. What is your opinion on this?

Reply:
Ok, makes sense. Let me do that.

Reply:
Great!